### PR TITLE
improve DecoyDatabase doc

### PR DESCRIPTION
**Pull request by bernt-matthias**
the tool does not shuffle/reverse the sequence
but only the peptides of a sequence.

fixes: https://github.com/OpenMS/OpenMS/issues/3634

Maybe someone can explain to me why for the tests data sequence

```
MVKQIESKTAFQEALDAAGDKLVVVDFSATWCGPCKMIKPFFHSLSEKYSNVIFLEVDVD
DCQDVASECEVKCMPTFQFFKKGQKVGEFSGANKEKLEATINELV
```

`MVKQ` stays the first 4 aminoacid in the result for method=reverse.

## Checklist
- [ ] Make sure that you are listed in the AUTHORS file
- [ ] Add relevant changes and new features to the CHANGELOG file
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] New and existing unit tests pass locally with my changes
- [ ] Updated or added python bindings for changed or new classes (Tick if no updates were necessary.)

### How can I get additional information on failed tests during CI
<details>
  <summary>Click to expand</summary>
If your PR is failing you can check out

- The details of the action statuses at the end of the PR or the "Checks" tab.
- http://cdash.openms.de/index.php?project=OpenMS and look for your PR. Use the "Show filters" capability on the top right to search for your PR number.
  If you click in the column that lists the failed tests you will get detailed error messages.

</details>

### Advanced commands (admins / reviewer only)
<details>
  <summary>Click to expand</summary>
  
- `/reformat` (experimental) applies the clang-format style changes as additional commit. Note: your branch must have a different name (e.g., yourrepo:feature/XYZ) than the receiving branch (e.g., OpenMS:develop). Otherwise, reformat fails to push.
- setting the label "NoJenkins" will skip tests for this PR on jenkins (saves resources e.g., on edits that do not affect tests)
- commenting with `rebuild jenkins` will retrigger Jenkins-based CI builds
  
</details>

---
:warning: Note: Once you opened a PR try to minimize the number of *pushes* to it as every push will trigger CI (automated builds and test) and is rather heavy on our infrastructure (e.g., if several pushes per day are performed).

### CODE REVIEW

**Comment by jpfeuffer** [comment]
Hi,

I think the explanation is:
`MVK` is a peptide. Rev with keeping ends is: `MVK`
`QIESK` is a peptide. Rev with keeping ends is: `QSEIK`

Appended again: `MVKQSEIK`

**Comment by bernt-matthias** [comment]
Ah, OK. So maybe we should add the keeping ends part to the docs? Just to be sure for `reverse` the first and last aminoacid stays? Is it the same for shuffle?

**Comment by jpfeuffer** [comment]
It should be in the param docs:
https://openms.de/develop_doxygen/html/UTILS_DecoyDatabase.html#:~:text=P%2CR%27%20here.-,keepPeptideNTerm,-true

**Comment by jpfeuffer** [comment]
But you can add the possibility of changing them to the main text.

File: src/utils/DecoyDatabase.cpp [hunk 1]
```diff
@@ -57,7 +57,7 @@ using namespace std;
 
   Decoy databases are useful to control false discovery rates and thus estimate score cutoffs for identified spectra.
 
-  The decoy can either be generated from reversed or shuffled sequences.
+  The decoy can either be generated by reversing or suffling each of the peptides of a sequence (as defined by a given enzyme).
```
Comment (bernt-matthias) [inline src/utils/DecoyDatabase.cpp:60]:
```suggestion
  The decoy can either be generated by reversing or shuffling each of the peptides of a sequence (as defined by a given enzyme). 
  For reversing the N and C terminus of the peptides are kept in position by default.
```

**Pull request merged**